local-deploy.sh

```diff
@@ -0,0 +1,223 @@
+    fi
+fi
+
+#----------------------------------------------------------------------------------
+
+echo
+echo "Validating nodejs..........................................-"
+
+if type -p nodejs; then
+    echo PASS: found nodejs executable in PATH
+    _nodejs=nodejs
+else
+    echo "FAIL: nodejs was not found on your machine"
+fi
+
+if [[ "$_nodejs" ]]; then
+    version=$("$_nodejs" -v 2>&1 | tr -d 'v')
+    vCompare_node=$(echo "${version}" 2>&1 | sed -e 's/[A-Za-z ._-]//g'| awk '{print substr($0,0,3)}')
+    echo version: "$version"
+    if (( $(echo "$vCompare_node" "$minNode" | awk '{print ($1 > $2)}') )); then
+        echo PASS: nodejs version is more than 4.0
+    else         
+        echo WARN: nodejs version is less than 4.0
+    fi
+fi
+if type -p npm; then
+    echo PASS: found npm executable in PATH
+    _npm=npm
+else
+    echo "FAIL: npm was not found on your machine"
+fi
+
+if [[ "$_npm" ]]; then
+    version=$("$_npm" -v )
+    echo version: "$version"
+    vCompare_npm=$(echo "${version}" 2>&1 | sed -e 's/[A-Za-z ._-]//g'| awk '{print substr($0,0,3)}')
+    if (( $(echo "$vCompare_npm" "$minNpm" | awk '{print ($1 > $2)}') )); then
+        echo PASS: npm version is more than 3.0
+    else         
+        echo WARN: npm version is less than 3.0
+    fi
+fi
+#----------------------------------------------------------------------------------
+
+
+echo
+echo "Validating scala..........................................-"
+
+if type -p scala; then
+    echo PASS: found scala executable in PATH
+    _scala=scala
+else
+    echo "FAIL: scala was not found on your machine."
+fi
+
+if [[ "$_scala" ]]; then
+    version=$("$_scala" -version 2>&1 | awk -F ' ' '{print $5}')
+    echo version: "$version"
+    vCompare_scala=$(echo "${version}" 2>&1 | sed -e 's/[A-Za-z ._-]//g'| awk '{print substr($0,0,3)}')
+    if (( $(echo "$vCompare_scala" "$minScala" | awk '{print ($1 > $2)}') )); then
+        echo PASS: scala version is more than 2.7
+    else         
+        echo WARN: scala version is less than 2.7 
+    fi
+fi
+
+#----------------------------------------------------------------------------------
+
+echo
+echo "Validating sbt..........................................-"
+
+if type -p sbt; then
+    echo PASS: found sbt directory in PATH
+    echo Checking fortis project sbt version.... "$(sbt sbtVersion)"
+else
+    echo "FAIL: sbt was not found on your machine."
+fi
+
+#----------------------------------------------------------------------------------
+
+echo
+echo "Validating cassandra..........................................-"
+
+if type -p cassandra; then
+    echo PASS: found sbt directory in PATH
+    _cassandra=cassandra
+else
+    echo "FAIL: cassandra was not found on your machine."
+fi
+
+
+if [[ "$_cassandra" ]]; then
+    version=$("$_cassandra" -v )
+    echo version: "$version"
+    vCompare_cassandra=$(echo "${version}" 2>&1 | sed -e 's/[A-Za-z ._-]//g'| awk '{print substr($0,0,3)}')
```

Comment:
Nit: bash has built-in support for string slicing, no need to go to awk for this:

```sh
vCompare_cassandra="${vCompare_cassandra:0:3}"
```